### ISSUE TITLE
optifabric for 1.20.1 has a bunch of 000000000 with Optifine.

### ISSUE THREAD

**Issue by DestructiveBurn** [Chocohead/OptiFabric#1172]
I am seeing a ton of 0000000000000000 in Minecraft where text should be. 

Current files.
fabric-api-0.86.1+1.20.1.jar
optifabric-1.13.25.jar
OptiFine_1.20.1_HD_U_I5.jar

I have tried.
fabric-api-0.86.0+1.20.1.jar
preview_OptiFine_1.20.1_HD_U_I6_pre3.jar

Screenshot
[https://i.gyazo.com/c5b9bc33479d270df7e40241f55acf17.jpg](https://i.gyazo.com/c5b9bc33479d270df7e40241f55acf17.jpg)

When is the full release coming out so I can use OptiFine_1.20.1_HD_U_I5.jar

**Comment by AuroraDoesCode** [comment]
Duplicate of #1123

**Comment by Uniqeues** [comment]
Yes the same issue here I really wanted optifine because its just better than sodium for me but I cant because of this 000000000 everywhere. I think its caused when used with fabric api because when I uninstall it everything goes back to normal.

**Comment by AuroraDoesCode** [comment]
> Yes the same issue here I really wanted optifine because its just better than sodium for me but I cant because of this [000000000](tel:000000000) everywhere. I think its caused when used with fabric api because when I uninstall it everything goes back to normal.

As said, duplicate of #1123

**Comment by DestructiveBurn** [comment]
I've already switched to Sodium and iris.  I am set.  I have noticed better performance too.   Goodbye, Optifabric.  You were handy but you're just too slow to update.   Sadly, that will be your downfall, but hey, at least you are another option if needed in the future!

**Comment by AuroraDoesCode** [comment]
> I've already switched to Sodium and iris. I am set. I have noticed better performance too. Goodbye, Optifabric. You were handy but you're just too slow to update. Sadly, that will be your downfall, but hey, at least you are another option if needed in the future!

Its literally a known issue, and has been for a couple of weeks. You can use Fabric API 0.85, or switch, up to you. If you're solved, you can close the pointless issue
